VMS/VMS.csproj

```diff
@@ -43,7 +43,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="wwwroot\img\RelatedActivities\" />
```

Comment:
this line was deleted, please revert it

Reply:
This line is somehow irrelevant, just ignore it.

Reply:
i got it